test_accounts/health_test_accounts.md

```diff
@@ -11,7 +11,9 @@ Use these test accounts to log into the ID.me sandbox environment that is connec
 ---
 
 **NOTE:**
-Details about Conditions associated to a patient can be found in [Test Patient Conditions](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/ramosan-add-patient-conditions/test_accounts/health_test_account_conditions.xlsx)
+
+ * Details about Conditions associated to a patient can be found in [Test Patient Conditions](https://github.com/department-of-veterans-affairs/vets-api-clients/blob/ramosan-add-patient-conditions/test_accounts/health_test_account_conditions.xlsx)
```

Comment:
I would add a period at the end of this sentence.

Reply:
Updated. Sorry Joe one day I will learn to use punctuation unprompted.

Reply:
No problem Andrew - it happens, that's what I'm here for!